scorex-basics/src/main/scala/scorex/account/Account.scala

```diff
@@ -7,7 +7,7 @@ import scorex.utils.ScorexLogging
 
 import scala.util.Try
 
-
+@SerialVersionUID(-5326597598126993189L)
```

Comment:
We don't use such approach, use BytesSerializable to serialize Address, it's a mistake to put Java object to db